<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Hompage Test Suite" duration-ms="23639" started-at="2018-12-29T16:03:49Z" finished-at="2018-12-29T16:04:13Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="23639" started-at="2018-12-29T16:03:49Z" finished-at="2018-12-29T16:04:13Z">
      <class name="com.mercuryTravel.tests.HomepageTests">
        <test-method status="PASS" signature="presetup()[pri:0, instance:com.mercuryTravel.tests.HomepageTests@1e88b3c]" name="presetup" is-config="true" duration-ms="515" started-at="2018-12-29T16:03:49Z" finished-at="2018-12-29T16:03:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- presetup -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.mercuryTravel.tests.HomepageTests@1e88b3c]" name="setup" is-config="true" duration-ms="16222" started-at="2018-12-29T16:03:49Z" finished-at="2018-12-29T16:04:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="verifyUserLoginToMercureyTravel()[pri:0, instance:com.mercuryTravel.tests.HomepageTests@1e88b3c]" name="verifyUserLoginToMercureyTravel" duration-ms="4666" started-at="2018-12-29T16:04:05Z" finished-at="2018-12-29T16:04:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Welcome, Saurabh 1] but found [Welcome, Saurabh]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Welcome, Saurabh 1] but found [Welcome, Saurabh]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.mercuryTravel.tests.HomepageTests.verifyUserLoginToMercureyTravel(HomepageTests.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserLoginToMercureyTravel -->
        <test-method status="PASS" signature="afterAMethod(org.testng.ITestResult)[pri:0, instance:com.mercuryTravel.tests.HomepageTests@1e88b3c]" name="afterAMethod" is-config="true" duration-ms="1614" started-at="2018-12-29T16:04:10Z" finished-at="2018-12-29T16:04:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyUserLoginToMercureyTravel status=FAILURE method=HomepageTests.verifyUserLoginToMercureyTravel()[pri:0, instance:com.mercuryTravel.tests.HomepageTests@1e88b3c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterAMethod -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:com.mercuryTravel.tests.HomepageTests@1e88b3c]" name="cleanup" is-config="true" duration-ms="1074" started-at="2018-12-29T16:04:12Z" finished-at="2018-12-29T16:04:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method status="PASS" signature="postCleanup()[pri:0, instance:com.mercuryTravel.tests.HomepageTests@1e88b3c]" name="postCleanup" is-config="true" duration-ms="1086" started-at="2018-12-29T16:04:13Z" finished-at="2018-12-29T16:04:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCleanup -->
      </class> <!-- com.mercuryTravel.tests.HomepageTests -->
    </test> <!-- Test -->
  </suite> <!-- Hompage Test Suite -->
</testng-results>
